code/game/objects/items/tanks/watertank.dm

```diff
@@ -439,7 +439,7 @@
 	user.update_inv_back() //for overlays update
 
 //Operator backpack spray
-/obj/item/watertank/operator
```

Comment:
operative, maybe.

Reply:
awwperative

Reply:
hurngigglejinggleringle

Reply:
`The operator keyword conflicted with older code such as the AbyssDragon.SET library. The word should once again be usable as a var name or path component.` 
http://www.byond.com/forum/?post=2299408

This can be reverted.